src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ToGerritRunListener.java

```diff
@@ -428,15 +429,19 @@ private String obtainFailureMessage(@Nullable GerritTriggeredEvent event,
                 // The filename may contain environment variables
                 filepath = envVars.expand(filepath);
 
-                // Check for ANT-style file path
-                FilePath[] matches = this.getMatchingWorkspaceFiles(build.getWorkspace(), filepath);
-                logger.debug("Found matching workspace files: {}", matches);
+                if (build instanceof AbstractBuild) {
```

Comment:
So is there still no way to solve this for a workflow? If so then the config page should reflect that.